cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -77,9 +77,17 @@ spec:
     - name: Age
       type: date
       JSONPath: .metadata.creationTimestamp
+    - name: CephImage
+      type: string
+      description: Ceph Image
+      JSONPath: .spec.cephVersion.image
```

Comment:
Note that #2818 will detect the version number such as 13.2.4. That would be good to add as well if you wouldn't mind waiting for that change to merge.